pkg/logger/log/accesslog.go

```diff
@@ -225,7 +225,7 @@ func AccessLog(req *restful.Request, resp *restful.Response, chain *restful.Filt
 }
 
 // getRequestBody will get the request body from Request object
-func getRequestBody(req *restful.Request, contentType string) string {
+func getRequestBody(req *restful.Request, contentType, requestURL string) string {
```

Comment:
Why must we consider whether it's a request or a response? Shouldn't we focus on the bytes directly?

Reply:
updated